js/config.js

```diff
@@ -15,7 +15,7 @@ _.defaultsDeep(systematic, {
     port: 8080,
   },
   test: {
-    file_pattern: '**/*tests.js'
+    file_pattern: 'src/**/*tests.js'
```

Comment:
kemar avait un souci dont je ne me rappelle plus. J'ai noté ça dans le commit. Mais là c'est juste que par défaut on chope tous les fichiers de test de node_modules, d'après metamatik.